conf/broker.conf

```diff
@@ -61,6 +61,9 @@ failureDomainsEnabled=false
 # Zookeeper session timeout in milliseconds
 zooKeeperSessionTimeoutMillis=30000
 
+# ZooKeeper operation timeout in seconds
```

Comment:
nit: Other config around this are in Ms so I would suggest to keep the TimeUnit in millis.